common/urls.py

```diff
@@ -1,8 +1,21 @@
-from django.urls import path
+from django.conf.urls import url
+from django.urls import path, include
 from common import views
-
+from config import settings
```

Comment:
:warning: `from django.conf import settings` https://docs.djangoproject.com/en/2.2/topics/settings/#using-settings-in-python-code